src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -177,4 +194,19 @@ private function getEndpoint(): ?string
     {
         return ($this->host ?: self::HOST).($this->port ? ':'.$this->port : '');
     }
+
+    private function castCustomHeader(string $value, string $type)
```

Comment:
Maybee, you could make last parameter optionnal `string $type = 'string'` to avoid `$convertConf[1] ?? 'string'`

Reply:
The main reason to use this construction is avoid to add second `string` parameter to HEADER_TO_MESSAGE members.
$convertConf[1] not always set and can trigger php warning if you don't provide default value `?? 'string'`.